synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -44,10 +44,34 @@ def get_exported_valuenode(self):
         """
         if self.param is None or "use" not in self.param.attrib.keys():
             return
+
+        # Full ID string format is like: [[file-path]#][canvas-id:][child-canvas-id:][child-canvas-id:]…value-node-id
+        # Refer here: https://github.com/synfig/synfig/issues/993
         layer = self.get_layer()
         canvas = layer.getparent()
         key = self.param.attrib["use"]
-        anim = canvas.get_def(key)
+        if ":" in key:  # Start searching from root canvas
+            canvas = settings.ROOT_CANVAS
```

Comment:
Wouldn't it be safer to check (later) whether first token keys[0 or 1].split(":") is empty?

There shouldn't be "something::other" in id, I guess.

Reply:
I am not sure what you are trying to say here. Do you mean that the file name can have a `:` inside it? If yes, then I will need to put this check later.
Is this what you mean here?

Reply:
Made the necessary changes.